pages/static/vendor/js/djaodjin-sidebar-gallery.js

```diff
@@ -111,29 +112,35 @@
             $(".droppable-image").droppable({
                 drop: function( event, ui ) {
                     var droppable = $(this);
+                    var source = ui.draggable.attr("src").toLowerCase();
                     if (droppable.prop("tagName") === "IMG"){
```

Comment:
OK for now. We might consider a way to generalize such that images or videos could be dropped in media placeholders.